src/Controllers/SchemaController.js

```diff
@@ -330,26 +330,26 @@ const injectDefaultSchema = ({className, fields, classLevelPermissions, indexes}
 
 const _HooksSchema =  {className: "_Hooks", fields: defaultColumns._Hooks};
 const _GlobalConfigSchema = { className: "_GlobalConfig", fields: defaultColumns._GlobalConfig }
-const _PushStatusSchema = convertSchemaToAdapterSchema(injectDefaultSchema({
```

Comment:
Don't need to call injectDefaultSchema here as it is called inside convertSchemaToAdapterSchema